terraform/modules/fhir/outputs.tf

```diff
@@ -0,0 +1,7 @@
+output "fhir_dataset_id" {
+  value = google_healthcare_dataset.dataset.name
```

Comment:
I think this should be `.id` instead of `.name`: https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/healthcare_dataset#attributes-reference

Reply:
Actually, if you look at what we are after name works.  ID will contain the full project_id/dataset_id/ etc...

id - an identifier for the resource with format projects/{{project}}/locations/{{location}}/datasets/{{name}}

vs

name - (Required) The resource name for the Dataset.